src/components/Claim/Link/Initial.view.tsx

```diff
@@ -635,6 +651,20 @@ export const InitialClaimLinkView = ({
                                             </span>
                                         </>
                                     )}
+                                    {errorState.errorMessage === 'offramp_lt_minimum' && (
+                                        <>
+                                            <label className=" text-h8 font-normal text-red ">
+                                                You can not claim links with less than ${MIN_CASHOUT_LIMIT} to your bank account.{' '}
+                                            </label>
+                                        </>
+                                    )}
+                                    {errorState.errorMessage === 'offramp_mt_maximum' && (
+                                        <>
+                                            <label className=" text-h8 font-normal text-red ">
+                                                You can not claim links with more than ${MAX_CASHOUT_LIMIT} to your bank account.{' '}
```

Comment:
tiny-nit: is this formatted as "$10100" ? not perfect but I VERY MUCH DOUBT anyone will ever run into this :laughing: